pwa/src/apiService/resources/dashboardCards.tsx

```diff
@@ -1,7 +1,16 @@
 import { Send } from "../apiService";
 import { AxiosInstance } from "axios";
 
-export type TEntity = "Action" | "Gateway" | "Cronjob" | "Endpoint" | "ObjectEntity" | "Entity" | "CollectionEntity" | "Organization";
+export type TEntity =
+  | "Action"
+  | "Gateway"
+  | "Cronjob"
+  | "Endpoint"
+  | "ObjectEntity"
+  | "Entity"
+  | "CollectionEntity"
+  | "Organization"
+  | "User";
```

Comment:
You will also need to set "User" in TDashboardCardType in the file useDashboardCards.ts